store/mongo/datastore_mongo.go

```diff
@@ -676,7 +676,7 @@ func (db *DataStoreMongo) IsArtifactUnique(ctx context.Context,
 
 	query := bson.M{
 		"$and": []bson.M{
-			bson.M{
+			{
```

Comment:
To me the original looks like the correct one, but if it works...

Reply:
I configured emacs to use gofmt with the `-s` option ("simplify"). It's exactly the same, but it helps making goreport happier :)